android/build.gradle.kts

```diff
@@ -1,8 +1,12 @@
+import org.jetbrains.kotlin.konan.properties.loadProperties
+
 plugins {
     id("com.android.application")
     kotlin("android")
 }
 
+val keystoreProperties = loadProperties("android/signing.properties")
```

Comment:
@kaungkhantsoe You might need to update your code: https://github.com/nimblehq/kmm-templates/pull/25/files#diff-74804964ccc306f732ea790fdd0a41b45f4d6a0426c9305a7c91e900e1303116R4